.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,45 @@
+name: Cypress E2E tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  cypress-run:
+    if: github.repository == 'redwoodjs/redwood'
+    runs-on: ubuntu-16.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install framework modules
+        run: yarn install --frozen-lockfile
+
+      - name: Create a RedwoodJS app
+        # NOTE: This path could be a problem if GitHub actions change something.
+        run: mkdir ./my-redwood-app && ./tasks/test-tutorial /home/runner/work/redwood/redwood/my-redwood-app
```

Comment:
You could make this more resilient by storing the path in a output variable and read it later when needed

Perhaps something like 
```
id: createpath
run: |
  project_path=$(mktemp -d -t redwood)
  ./task/test-tutorial $project_path
  echo "::set-output name=project_path::$project_path"
```

Then, when you need to reference it later you can do
```
working-directory: ${{ steps.createpath.outputs.project_path }}
```
or
```
env:
  CYPRESS_RW_PATH: "${{ steps.createpath.outputs.project_path }}"
```

Reply:
Oooh nice!